File: Frontend/marcador-pontos/src/entities/partida.ts
export class Partida {
    id: number; 
    dataPartida: string; 
    recordeQuebrado: boolean;
    pontuacao: number; 

    constructor(
        id: number,
        dataPartida: string,
        recordeQuebrado: boolean = false,
        pontuacao: number
    ) {
        this.id = id;
        this.dataPartida = dataPartida;
        this.recordeQuebrado = recordeQuebrado;
        this.pontuacao = pontuacao;
    }
}

File: Backend/MarcadorPontos/Domain/Entity/Partida.cs
namespace Domain.Entity
{
    public class Partida
    {
        #region Properties
        public uint Id { get; set; }
        public string DataPartida { get; set; }
        public bool RecordeQuebrado { get; set; }
        public ushort Pontuacao { get; set; }
        public DateOnly DataPartidaConvertida { get; set; }
        #endregion

        #region Constructors
        public Partida()
        {
        }
        #endregion
    }
}
